### PR TITLE
map prompt for duplicate creation

### PR DESCRIPTION
**Pull request by mfordjody**

### CODE REVIEW

**Comment by mfordjody** [comment]
https://github.com/opencurve/curveadm/issues/177

File: internal/task/task/bs/create_volume.go [hunk 1]
```diff
@@ -76,6 +77,13 @@ func NewCreateVolumeTask(curveadm *cli.CurveAdm, cc *configure.ClientConfig) (*t
 	subname := fmt.Sprintf("hostname=%s image=%s", hc.GetHostname(), cc.GetContainerImage())
 	t := task.NewTask("Create Volume", subname, hc.GetSSHConfig())
 
+	if len(cc.GetContainerImage()) == 0 {
```
Comment (Wine93) [inline internal/task/task/bs/create_volume.go:80]: Actually, there is a default value for client container image, so maybe it's an useless branch.
BTW: 
* you should add a step to do checking, maybe `step.Lamba` is a good choice.
* what is the purpose of those changes.

File: internal/task/task/bs/create_volume.go [hunk 2]
```diff
@@ -76,6 +77,13 @@ func NewCreateVolumeTask(curveadm *cli.CurveAdm, cc *configure.ClientConfig) (*t
...
+		if err != nil {
+			return t, nil
+		}
+	} else {
```
Comment (tsonglew) [inline internal/task/task/bs/create_volume.go:84]: We should prompt based on the actual volume size and the expected, rather than the container image.

**Comment by mfordjody** [comment]
I have been too busy lately and haven't taken the time to process this request. I will handle it in the near future :)

**Comment by mfordjody** [comment]
```
[root@master ~]# lsblk
NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
sda           8:0    0   50G  0 disk
├─sda1        8:1    0    1G  0 part /boot
└─sda2        8:2    0   49G  0 part
  ├─cs-root 253:0    0   46G  0 lvm  /
  └─cs-swap 253:1    0    3G  0 lvm  [SWAP]
sdb           8:16   0   20G  0 disk
sdc           8:32   0   20G  0 disk
sdd           8:48   0   20G  0 disk
sr0          11:0    1 10.7G  0 rom
```
```
[root@master ~]# curveadm format -f format.yaml
Start Format Chunkfile Pool: [SKIP]
  + host=master  device=/dev/sdb  mountPoint=/data/chunkserver0  usage=50% [1/1] [SKIP]
  + host=master  device=/dev/sdc  mountPoint=/data/chunkserver1  usage=50% [1/1] [SKIP]
  + host=master  device=/dev/sdd  mountPoint=/data/chunkserver2  usage=50% [1/1] [SKIP]
```
```
[root@master ~]# curveadm map curveuser1:/test --host master -c client.yaml --create --size 20GiB
Check Kernel Module: [OK]
  + host=master  module=nbd [1/1] [OK]

Start NEBD Service: [ERROR]
  + hostname=11.0.1.131  image=opencurvedocker/curvebs:v1.2 [0/1] [ERROR]
---
Error-Code: 420000
Error-Description: volume already mapped
Error-Clue: volume: curveuser1:/test
How to Solve:
  * Website: https://github.com/opencurve/curveadm/wiki/errno4#420000
  * Log: /root/.curveadm/logs/curveadm-2023-05-23_03-24-35.log
  * WeChat: opencurve_bot
```
```
[root@master ~]# curveadm client status
Id  Kind  Host  Container Id  Status  Aux Info
--  ----  ----  ------------  ------  --------
```

**Comment by caoxianfei1** [comment]
@mfordjody anything changed?

**Comment by mfordjody** [comment]
> @mfordjody有什麼改變嗎？

I am currently unable to conduct testing because I have encountered the above situation

**Comment by caoxianfei1** [comment]
> > @mfordjody有什麼改變嗎？
> 
> I am currently unable to conduct testing because I have encountered the above situation @caoxianfei1

you can delete the the container and retry it.

**Comment by mfordjody** [comment]
I redeploy a new testing environment, but I always feel like someone has changed the chunkserver

**Comment by mfordjody** [comment]
Requires longer modification time

**Pull request closed without merging**